package.json

```diff
@@ -12,6 +12,8 @@
     "@mui/material": "^5.10.7",
     "@mui/styles": "^5.10.7",
     "@mui/x-date-pickers": "^5.0.3",
+    "add-to-calendar-button": "^2.2.9",
+    "add-to-calendar-button-react": "^2.2.7",
```

Comment:
I wouldn't think you need both of these. Can you chose one and only use that one?

Reply:
Sure! The second one is what we only need.